### DISCUSSION TITLE
Authentication error

### DISCUSSION THREAD

**Discussion by sahil55c** [community/community#119511] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Question

Body
Hi,

After quite a while of not using it, I wanted to relogin to one (of two) GitHub accounts I have. I am greeted with a text stating I will be sent an email with a code. The email however does not arrive:
"We just sent your authentication code via email to . The code will expire at 3:16PM CEST."

My email account is correct, it is working, I tried multiple times over the past hours.
Does anybody have an idea what could be happening?

I'd have to add that I found a string of emails from last December and Januray urging me to enroll 2FA. These also stated "On January 19th, 2024 at 00:00 (UTC) your account will be required to have 2FA for authentication. If you have not yet enrolled by that date, your ability to access GitHub.com will be limited until you finish the enrollment process." So even though that's a while back there is the guarantee I will be able to get back in, and presumably be forced to enroll 2FA.

**Comment by im-divxnshh** [comment]
same problem :(

**Comment by PratishthaGaur** [comment]
Same problem. 
Is there any way to reach GitHub Support?

**Comment by jcb2802** [comment]
I forgot password to my account and didn’t receive any reset password e-mail.

**Comment by ettaboyle** [comment]
The best (and only) place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). We cannot respond to any account specific questions in this public forum. 

Please open a ticket with them :)
